doc/calling-javascript.md

```diff
@@ -307,12 +306,14 @@ access on Scala's side as well, but it is not required to use these names.
 ## JavaScript classes
 
 It is also possible to define JavaScript *classes* as Scala classes inheriting,
-directly or indirectly, from `js.Object`. The only difference compared to
+directly or indirectly, from `js.Any`. The main difference compared to
```

Comment:
Please somehow add `js.Object` here, so that people reading just this know of its existence.